### PR TITLE
[MOSIP-34574] Implemented monitoring and logging stack using external…

### PR DESCRIPTION
**Pull request by Mohanraj209**
… helm charts.

### CODE REVIEW

File: logging/README.md [hunk 1]
```diff
@@ -3,14 +3,27 @@
...
 ```sh 
 ./install.sh
 ```
+Note: Before running the `install.sh` script please make sure the `values.yaml` is updated. 
 
-## Install Rancher FluentD system
-1. Install Logging from Apps and marketplace within the Rancher UI.
-1. Select Chart Version `103.1.1+up4.4.0` from Rancher console -> _Apps & Marketplaces_.
+## Install logging operator. 
+There are two ways to install monitoring app as given below:
```
Comment (ckm007) [inline logging/README.md:13]: There are two ways to install **logging** app as given below:

File: logging/install.sh [hunk 2]
```diff
@@ -11,15 +11,36 @@ NS=cattle-logging-system
...
-helm repo update
+function installing_logging() {
+  echo Updating helm repos
+  helm repo add bitnami https://charts.bitnami.com/bitnami
+  helm repo add banzaicloud-stable https://kubernetes-charts.banzaicloud.com
+  helm repo update
 
-echo Installing Bitnami Elasticsearch and Kibana
-helm -n $NS install elasticsearch mosip/elasticsearch -f es_values.yaml --version 17.9.25 --wait
+  echo Installing Bitnami Elasticsearch and Kibana
```
Comment (ckm007) [inline logging/install.sh:20]: kibana istio objects

File: logging/README.md (hunk 1 above)
Comment (Mohanraj209) [reply logging/README.md:13]: updated

File: logging/install.sh (hunk 2 above)
Comment (Mohanraj209) [reply logging/install.sh:20]: updated

File: logging/delete.sh [hunk 3]
```diff
@@ -1,19 +1,33 @@
...
-        break
-      else
-        break
-    fi
-done
+
+function deleting_logging() {
+  while true; do
+      read -p "Are you sure you want to delete ALL logging helm charts from $KUBECONFIG cluster?(Y/n) " yn
+      if [ $yn = "Y" ]
```
[the remark this answers is not in the document]
Comment (Mohanraj209) [reply logging/delete.sh:14]: updated

**Review by ckm007 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**